ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/validation/ValidationResults.scala

```diff
@@ -28,15 +31,19 @@ object ValidationResults {
         warnings.invalidNodes.combine(other.warnings.invalidNodes)
       ),
       variableTypes ++ other.variableTypes,
+      parameters ++ other.parameters,
       typingInfo ++ other.typingInfo
     )
 
     def withTypes(variableTypes: Map[String, Map[String, TypingResult]]): ValidationResult
-      = copy(variableTypes = variableTypes)
+    = copy(variableTypes = variableTypes)
```

Comment:
Can you left `=` on the end of the line above (I know that it was like this before your change)

Reply:
Intellij formatter does this... I also find it a bit weird - but if we want to be consistent we'd better create own format template...

Reply:
Strange when I've changed this code to this:
```
    def withTypes(variableTypes: Map[String, Map[String, TypingResult]]): ValidationResult = 
       copy(variableTypes = variableTypes)
```
Intellij doesn't try to fix it. What version you are using? To be clear, I don't have a problem with indention but with place where `=` is located: end of line vs beginning of line.